### PR TITLE
Feature/mini 1789: Allow inline video playback

### PR DESCRIPTION
**Pull request by Climbatize**
# Description
Currently mini apps can't display videos inline when <video playsinline> is explicitly coded in the html script.
We set the inline configuration to true into the SDK to allow this behavior

## Links
[MINI-1789](https://jira.rakuten-it.com/jira/browse/MINI-1789)

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

Screen Recording 2020-09-29 at 17 17 03

### CODE REVIEW

File: MiniApp/Classes/Display/MiniAppWebView.swift [hunk 1]
```diff
@@ -10,6 +10,8 @@ internal class MiniAppWebView: WKWebView {
         config.preferences.javaScriptEnabled = true
         config.preferences.setValue(true, forKey: "allowFileAccessFromFileURLs")
         config.setURLSchemeHandler(URLSchemeHandler(versionId: versionId), forURLScheme: schemeName)
+        config.allowsInlineMediaPlayback = true
+        config.allowsPictureInPictureMediaPlayback = true
```
Comment (corycaywood) [inline MiniApp/Classes/Display/MiniAppWebView.swift:14]: Does the mini app need to add anything to use picture-in-picture? Or just adding `playsinline` will enable both?

**Review by corycaywood — COMMENTED** [review]
Looks good 👍  But looks like you reverted the `js-miniapp` sub-module to an old commit, so please either change this back or update it to the latest commit.

Also, could you add this to the change-log? I think we won't include this in v2.2.0, so maybe create a new section for v2.3.0?

File: MiniApp/Classes/Display/MiniAppWebView.swift (hunk 1 above)
Comment (Climbatize) [reply MiniApp/Classes/Display/MiniAppWebView.swift:14]: No, this is the default behavior for all videos into a basic web view. In fact it seems like adding `playsinline` disables PiP. I just added this to the config to consolidate the default value, so we have a consistent behavior in Mini App SDK if the default value of this parameter was ever changed by Apple in a future update.

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,6 +26,10 @@
 - **Feature:** Added sample implementation for Sharing the message from Mini app
 - **Feature:** Added sample app implementation to revoke/manage the custom permissions for the list of downloaded mini apps
 
+### 2.x
+
+***SDK***
+-**Fixed:** support of `playsinline` instruction of  `Video` html tag
```
Comment (ts-leojoseph-r) [inline CHANGELOG.md:32]: I think this has to be moved to the top `2.x.0`. That has our next release updates

**Review by ts-leojoseph-r — APPROVED** [review]
Looks good to me :+1: Just one comment. Please check

File: CHANGELOG.md (hunk 2 above)
Comment (corycaywood) [reply CHANGELOG.md:32]: Sorry, this should be move to a separate release version above the other `2.x.0` - this change won't be included in the `2.1.0` release.

Comment (Climbatize) [reply CHANGELOG.md:32]: men! I thought I was safe with 2.x as it's just not even a version number XD

**Review by corycaywood — APPROVED** [review]
👍

**Pull request merged**